src/db/sqlite/lrsql/sqlite/sql/delete.sql

```diff
@@ -59,3 +59,13 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope
+
+-- :name delete-reaction!
+-- :command :execute
+-- :result :affected
+-- :doc (Soft) delete a reaction.
```

Comment:
good call on soft delete for statement FK

Reply:
Yeah I think it makes sense, even if it is the only soft delete in the system (not counting voids which are kind of one)